src/store/modules/actions/user.js

```diff
@@ -77,7 +79,7 @@ export async function resumeCanceledSubscription({ dispatch }) {
 
 export async function updatePreferences(
   { dispatch, commit },
```

Comment:
Can add getters here, then you can use `getters.getUserInfo` later without import cycle
```suggestion
  { dispatch, commit, getters },
```